generators/app/templates/src/app/core/authentication/__auth.authentication.service.ts

```diff
@@ -91,4 +93,19 @@ export class AuthenticationService {
     }
   }
 
+  /**
```

Comment:
I'm wondering about this one: while I'm not found of public shared properties, since there is no need for extra work in these getters/setters I would say it's simpler to juste expose `redirectUrl` publicly (without a getter/setter) to follow the KISS principle.